recipes/taocpp-json/all/test_package/example.cpp

```diff
@@ -53,4 +53,4 @@ int main( int /*unused*/, char** /*unused*/ )
    tao::json::validate_integer< tao::json::events::discard, 10, 40 > consumer;
    tao::json::events::from_value( consumer, v );
    return 0;
-}
\ No newline at end of file
+}
```

Comment:
```suggestion
}

```

Reply:
I did add an endline in the latest commit, but I think it skipped the check? It didn't produce a report.

Reply:
you are right!